packages/plugins/plugin-sheet/src/components/Toolbar/Toolbar.tsx

```diff
@@ -235,12 +240,17 @@ const Alignment = () => {
   );
 };
 
-const styleOptions: ButtonProps<StyleValue>[] = [{ value: 'highlight', icon: 'ph--highlighter--regular' }];
+const styleOptions: ButtonProps<StyleValue>[] = [
+  { value: 'highlight', icon: 'ph--highlighter--regular' },
+  { value: 'softwrap', icon: 'ph--paragraph--regular' },
+];
 
 const Styles = () => {
   const { cursor, model } = useSheetContext();
   const { onAction } = useToolbarContext('Styles');
   const { t } = useTranslation(SHEET_PLUGIN);
+  // TODO(thure): Why is neither model.sheet nor model.sheet.ranges not usable as a memo dependency?
```

Comment:
What happens without this?

Reply:
The toolbar item isn’t reactive even though the range is added.

Reply:
May just remove the `useMemo`…